src/asset-swapper/utils/market_operation_utils/index.ts

```diff
@@ -552,6 +552,8 @@ export class MarketOperationUtils {
 
         // If there is no optimal path AND we didn't return a MultiHop quote, then throw
         if (optimalPath === undefined) {
+            //temporary logging for INSUFFICIENT_ASSET_LIQUIDITY
```

Comment:
since we are already logging is there any context to log along with?  (the first param)